tags.txt

```diff
@@ -0,0 +1,3 @@
+2.10.1
```

Comment:
these are the version numbers of the 3 different images? not having labels or anything isn't very scalable. If this is the way we want to do it, I'd put them in a json where the tags are in an array so if an image is added/removed we don't have to remember or skip lines

```json
{
"aks/helloworld": ["1.0.0", "1.0.1"]
}
```
etc.

That way we can keep track of what versions have existed. And if not, they can just be strings of the next version. A tool like `jq` would replace `sed` in the bash commands